src/main/java/net/atos/zac/app/zoeken/converter/RESTZaakZoekObjectConverter.java

```diff
@@ -64,19 +62,11 @@ public RESTZaakZoekObject convert(final ZaakZoekObject zoekItem) {
         restZoekItem.statusToelichting = zoekItem.getStatusToelichting();
         restZoekItem.indicaties = zoekItem.getZaakIndicaties();
         restZoekItem.rechten = restRechtenConverter.convert(policyService.readZaakRechten(zoekItem));
-        restZoekItem.betrokkenen = new EnumMap<>(RolType.OmschrijvingGeneriekEnum.class);
```

Comment:
Are you sure this is a correct change? An EnumMap is preferable I think because it is typesafe, in contrast to a HashMap.

Reply:
Ah, understand it now. This whole story is about not using those omschrijving generiek fields anymore but switch to the 'omschrijvingen' which are strings by nature.